src/printer/index.js

```diff
@@ -736,6 +736,20 @@ function genericPrint(path, options, print) {
       ]);
     }
 
+    case "Nonlocal": {
+      return groupConcat([
+        "nonlocal ",
```

Comment:
Hm, seems like we'd want an `escapedLine` (or something with parens) here. Similarly, would be nice to see a test with multiple names, with and without needing to wrap.